config.d.ts

```diff
@@ -40,7 +40,7 @@ export type Config = {
      *
      * This is useful for settings options like `channel`, `headless`, `executablePath`, etc.
      */
-    launchOptions?: playwright.BrowserLaunchOptions;
+    launchOptions?: playwright.LaunchOptions;
```

Comment:
How come TSC did not complain? Are we missing tsc linter on the bots?

Reply:
`/config.d.ts` is not included in the `tsconfig.json` - only `src` is.

Reply:
https://github.com/microsoft/playwright-mcp/pull/386